src/main/java/graphql/language/Field.java

```diff
@@ -22,11 +27,13 @@
     private final List<Argument> arguments;
     private final List<Directive> directives;
     private final SelectionSet selectionSet;
+    private final Map<String, String> additionalData;
```

Comment:
I thought it would be `Map<String,Object>` for addtional data and hence any data keyed by String key

Also why just on fields.  Why cant every node have additional data?  We could put it on `NodeBuilder` to help with the implementation